### PR TITLE
Update data-semantic-conventions.md

### PR DESCRIPTION
**Pull request by davebarda**

### CODE REVIEW

**Comment by davebarda** [comment]
I signed it

File: specification/data-semantic-conventions.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Depending on the protocol and the type of operation, additional information
 is needed to represent and analyze a span correctly in monitoring systems. It is
 also important to unify how this attribution is made in different languages.
 This way, the operator will not need to learn specifics of a language and
-telemetry collected from multi-language micro-service can still be easily
+telemetry collected from polyglot micro-services environment can still be easily
```
Comment (SergeyKanzhelev) [inline specification/data-semantic-conventions.md]:
```suggestion
telemetry collected from polyglot (multi-language) micro-services environment can still be easily
```

**Comment by SergeyKanzhelev** [comment]
I like how the sentence sounds using the polyglot better. However if it causes question as per  @arminru comment we may have both - polyglot and multi-language in parenthesis.

**Comment by davebarda** [comment]
Hey, any updates regarding to the PR?

**Comment by davebarda** [comment]
I signed it

File: specification/data-semantic-conventions.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Depending on the protocol and the type of operation, additional information
 is needed to represent and analyze a span correctly in monitoring systems. It is
 also important to unify how this attribution is made in different languages.
 This way, the operator will not need to learn specifics of a language and
-telemetry collected from multi-language micro-service can still be easily
+telemetry collected from polyglot (multi-language) micro-services environment can still be easily
 correlated and cross-analyzed.
```
Comment (arminru) [inline specification/data-semantic-conventions.md]:
```suggestion
telemetry collected from polyglot (multi-language) micro-service environments
can still be easily correlated and cross-analyzed.
```

**Comment by SergeyKanzhelev** [comment]
seems to be the feedback was addressed. Merging

**Pull request merged**